cli_tools/common/utils/string/string_utils_test.go

```diff
@@ -48,14 +48,14 @@ func TestCombineStringSlices(t *testing.T) {
 			want: []string{"WINDOWS"},
 		},
 		{
-			s1:   []string{"SECURE_BOOT"},
+			s1:   []string{"MULTI_IP_SUBNET"},
```

Comment:
Is `MULTI_IP_SUBNET` here just to have something in the list so it's not empty?

Reply:
All of the instances of `SECURE_BOOT` were opaque references used in these utility tests. So I swapped them for a feature that wasn't deprecated.